src/main/scala/is/hail/methods/Aggregators.scala

```diff
@@ -17,6 +17,75 @@ import scala.reflect.ClassTag
 
 object Aggregators {
 
+  def buildVariantAggregationsByKey(vsm: VariantSampleMatrix, nKeys: Int, keyMap: Array[Int], ec: EvalContext): Option[(RegionValue) => Iterator[() => Unit]] =
```

Comment:
I'm having trouble understanding the Option and Iterator. Would it be be possible for the return type to be `(RegionValue) => Array[() => Unit]`? Or even just one function that does work on all the aggregations?